src/Documents/Import/PageImporter.php

```diff
@@ -51,4 +69,26 @@ public function readYamlFileAndSetParameters(string $filename, array $params = [
 
```

Comment:
Kann die Methode `readYamlFileAndSetParameters` weg? Bei mir sieht die unused aus

Reply:
No, this method will be used in other bundles already (e.g. trinity-pimcore-elastic-bundle)
I've tried to avoid breaking changes - keep in mind.